### ISSUE TITLE
Error Pytorch backend training - Expected object of scalar type

### ISSUE THREAD

**Issue by PiyushM1** [Tessellate-Imaging/monk_v1#92]
I used the pytorch backend to train 2 models with number of classes as 49 and 16. When I started training the third model with similar conditions, but with 174 classes, I received [this error](https://stackoverflow.com/questions/56741087/how-to-fix-runtimeerror-expected-object-of-scalar-type-float-but-got-scalar-typ?rq=1). 

Attaching the screenshots. I believe there's nothing I can do because of the abstraction from using monk. Please suggest a fix.
![Screenshot 2020-08-04 at 2 26 01 PM](https://user-images.githubusercontent.com/52419808/89274799-09085680-d65f-11ea-9a7f-a66f8eff9a03.png)
![Screenshot 2020-08-04 at 2 26 16 PM](https://user-images.githubusercontent.com/52419808/89274795-086fc000-d65f-11ea-9f58-7cec003a6fcd.png)
![Screenshot 2020-08-04 at 2 26 30 PM](https://user-images.githubusercontent.com/52419808/89274784-04dc3900-d65f-11ea-8abb-097b577575b5.png)

**Comment by abhi-kumar** [comment]
Share the dataset details and your code. Please check if the training csv has the right labels.

**Comment by PiyushM1** [comment]
I had earlier trained the same model with a keras backend, it had worked that time, so I don't think the labels are incorrect. Moreover, the labels for the other two classifiers are also prepared in the same way. 

Code: https://colab.research.google.com/drive/1Qt0UvKNbdUTwXbbh544bjJ6_7vhvol_f?usp=sharing

Check the Model Classifier part of the code (2nd classifier)

**Comment by abhi-kumar** [comment]
Your csv file for model classification contains a mix of strings and integers as classes, make sure every label is a string; and there are no white spaces in class names;

Suggestion 1- if your class name is an integer, say 30, change it to "cls_30"
Suggestion 2 - if you class name has white space, say "sonata hybrid", change it to "sonata_hybrid"

**Comment by PiyushM1** [comment]
Thank you. I replaced the spaces with underscores and it worked.
